openqa-prepare-mm-setup

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env perl
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+use strict;
+use autodie ':all';
+
+system("zypper in -y os-autoinst-openvswitch");
+system("systemctl enable --now os-autoinst-openvswitch");
+
+my $bridge = 1;
+my $openvswitch = "/etc/sysconfig/os-autoinst-openvswitch";
+open my $fh, '>', $openvswitch or die "Failed to open $openvswitch for writing!";
+print $fh "OS_AUTOINST_USE_BRIDGE=br$bridge";
+close($fh);
+
+system("ovs-vsctl add-br br$bridge");
+print("Added bridge $bridge");
+
+my $bridge_file = "/etc/sysconfig/network/ifcfg-br$bridge";
+my $ip = "10.0.2.2/15";
+my $config = "
+BOOTPROTO='static'                                                                                                         
+IPADDR='$ip'                                                                                                       
+STARTMODE='auto'                                                                                                           
+ZONE=trusted                                                                                                               
+OVS_BRIDGE='yes'                                                                                                           
+PRE_UP_SCRIPT=\"wicked:gre_tunnel_preup.sh\"
+";
+
+for my $i (0..147) {
+    $config .= "OVS_BRIDGE_PORT_DEVICE_$i='tap$i'\n";
+
+    my $tapconfig = "
+    BOOTPROTO='none'
+    IPADDR=''
+    NETMASK=''
+    PREFIXLEN=''
+    STARTMODE='auto'
+    TUNNEL='tap'
+    TUNNEL_SET_GROUP='nogroup'
+    TUNNEL_SET_OWNER='_openqa-worker'
+    ZONE=trusted
+    ";
+
+    my $tap_file = "/etc/sysconfig/network/ifcfg-tap$i";
+    open my $fh, '>', $tap_file or die "Failed to open $tap_file for writing!";
+    print $fh $tapconfig;
+    close($fh);
+}
+
+open my $fh, '>', $bridge_file or die "Failed to open $bridge_file for writing!";
+print $fh $config;
+close($fh);
+print("Config written!");
+
+my $gre = "/etc/wicked/scripts/gre_tunnel_preup.sh";
+open my $fh, '>', $gre or die "Failed to open $gre for writing!";
+
+my $gre_config = '
+#!/bin/sh
+action="$1"
+bridge="$2"
+ovs-vsctl set bridge $bridge stp_enable=true
+';
+
+my %workers = (openqaworker1 => '192.168.112.6',
+               openqaworker7 => '192.168.112.12',
+               openqaworker4 => '192.168.112.7',
+               openqaworker19 => '192.168.112.17',
+              );
+my $device = 0;
+my $this_worker = qx(hostname -i);
+chomp $this_worker;
+while (my ($worker, $ip) = each %workers) {
+    # Don't put the machine itself here
+    if ($this_worker ne $ip) {
```

Comment:
Maybe reverse the logic via `next if $ip eq $this_worker;  # don't put the machine itself here`.